Docker-compose.yml

```
@@ -0,0 +1,31 @@
+version: "3"
+
+services:
+  development:
+    env_file:
+        - .env
+    build: 
+      context: .
+      target: development
+    
+    volumes: 
+        - type: bind
+          source: ./todo_app
+          target: /todo_app/todo_app
+    ports:
+        - "5000:5000"
+
+  production:
+    env_file:
+        - .env
+    build: 
+      context: .
+      target: production
+      
+    .volumes: 
+        - type: bind
+          source: ./todo_app
+          target: /todo_app/todo_app
+  
+    ports:
+        - "5000:5000" 
```

Worth noting that with both apps pointing to port 5000 on the host, only one can be run at a time. That's fine if you want people to run them one at a time (e.g. with `docker-compose up production`) but we could switch this to an alternative port so that both can be run at the same time - moving the prod setup to the below would mean we leave dev available on your computer's 5000 while prod is available on 5001
```suggestion
        - "5001:5000" 
```